streams/src/main/java/org/apache/kafka/streams/state/internals/RocksDBStore.java

```diff
@@ -444,7 +444,7 @@ private void closeOpenIterators() {
         @Override
         public synchronized boolean hasNext() {
             if (!open) {
-                throw new InvalidStateStoreException(String.format("RocksDB store %s has closed", storeName));
```

Comment:
nit: how about `the underlying RocksDB store for this iterator has closed`?

Reply:
`open` does not indicate if the underlying store is closed or not -- it only indicates if the iterator was closed.

(Of course, if the store gets closed, the iterator will be closed, too, but if `open==false` we don't know if the store is closed or not.)